bower.json

```diff
@@ -20,16 +20,15 @@
     "package.json"
   ],
   "dependencies": {
-    "purescript-maybe": "^3.0.0",
-    "purescript-integers": "^3.0.0",
-    "purescript-strings": "^3.5.0"
+    "purescript-maybe": "^4.0.0",
+    "purescript-integers": "^4.0.0",
+    "purescript-strings": "^4.0.0"
   },
   "devDependencies": {
-    "purescript-integers": "^3.0.0",
-    "purescript-console": "^3.0.0",
-    "purescript-assert": "^3.0.0",
-    "purescript-quickcheck": "^4.0.0",
-    "purescript-quickcheck-laws": "^3.0.0",
```

Comment:
Hasn't been updated in a long time. Can make a note that this is on hold for it or leave out for now.